src/test/test_pidfile.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash -e
+
+#
+# Generic read/write from object store test
+#
+
+# Includes
+source ../qa/workunits/ceph-helpers.sh
+
+function run() {
+    local dir=$1
+    shift
+
+    export CEPH_MON="127.0.0.1:17108"
+    export CEPH_ARGS
+    CEPH_ARGS+="--fsid=$(uuidgen) --auth-supported=none "
+    CEPH_ARGS+="--mon-host=$CEPH_MON "
+
+    local funcs=${@:-$(set | sed -n -e 's/^\(TEST_[0-9a-z_]*\) .*/\1/p')}
+    for func in $funcs ; do
+        $func $dir || return 1
+    done
+}
+
+
+function TEST_normal() {
+    local dir=$1
+    setup $dir 
+    run_mon $dir a
+    run_osd $dir 0
+    teardown $dir 
```

Comment:
i am not sure if we need this test, as we use `run_mon` and `run_osd` in exactly the same way in quite a few of other tests here.